### ISSUE TITLE
MythX internal error

### ISSUE THREAD

**Issue by shashank-srikant** [ConsenSys/mythx-developer-support#16]
I run into a MythX internal error when trying to analyze a sample contract.

![mythx_err](https://user-images.githubusercontent.com/14936946/54087703-eec42f00-432b-11e9-8ba3-190261d98b03.png)

Suggestions on what's going wrong?

## Steps to reproduce
- This is the truffle project i am trying to run -
https://github.com/shashank-srikant/mythril-line-numbers

- Commands to run
```
npm install
./node_modules/.bin/truffle run verify StandardToken
```
- Versions
solc: Version: 0.4.24+commit.e67f0147.Linux.g++
truffle-security: 1.1.0
remix-lib: 0.4.1
mythril-classic: 0.20.0

**Comment by rocky** [comment]
This should be fixed when the next release 1.3.0 comes out. I think there's a way to watch releases using the "Watch" button in truffle-security.

**Comment by rocky** [comment]
... and release 1.3.0 is now out.
